legion/bin/legionctl

```diff
@@ -30,6 +30,8 @@ if __name__ == '__main__':
     legion.cli.configure_logging(args)
     root_logger = logging.getLogger()
 
+    root_logger.debug('CLI parameters: %s', args)
```

Comment:
We're not using % formatting, event for logs. We're using `format`

Reply:
We use such formatting in a lot of places. pylint have warning for case of `format`  https://docs.python.org/3/howto/logging.html#optimization